.travis.yml

```diff
@@ -78,3 +78,18 @@ matrix:
     - pip install -r requirements.txt
     - pip install -r cirq/contrib/contrib-requirements.txt
     script: check/doctest -q
+  - stage: deploy
+    os: linux
+    env: NAME=dev-release
+    python: '3.7'
+    script: export CIRQ_DEV_VERSION=`dev_tools/packaging/generate-dev-version-id.sh`
```

Comment:
Nit Nit: I like `$(...)` instead of backticks.

Reply:
changed